docs/spectrum/fitting.rst

```diff
@@ -8,20 +8,84 @@ Spectral Fitting
 
 .. currentmodule:: gammapy.spectrum
 
+In the following you will see how to fit spectral data in OGIP format. The
+format is described at :ref:`gadf:ogip`. An example dataset is available in the
+`gammapy-extra repo <https://github.com/gammapy/gammapy-extra/tree/master/datasets/hess-crab4_pha>`_.
 
-This section explains the classes used to perform a spectral git with Gammapy. For a introductory tutorial using the ``gammapy-spectrum`` command line tool see :ref:`tutorials-gammapy-spectrum`
+Getting Started
+===============
 
-TODO: explain class interface
+The following example shows how to fit a power law simultaneously to two
+simulated crab runs using the `~gammapy.spectrum.SpectrumFit` class.
 
+.. code-block:: python
 
-.. _background-estimation:
+    import astropy.units as u
+    from gammapy.spectrum import (
+        SpectrumObservation,
+        SpectrumObservationList,
+        SpectrumFit,
+        models,
```

Comment:
I'd prefer using this import convention in examples / docs:
```python
from gammapy.spectrum.models import PowerLaw
```
because `models` is such a general term that it's non-optimal to have it refer to a module in the script namespace.